### ISSUE TITLE
Why doesn't vscode have a embedded version?

### ISSUE THREAD

**Issue by velara3** [microsoft/vscode#237026]
Is there any plan to have vscode be embedded in web pages or editors? 

For example, if you go to StackOverflow or Github or other sites and want to display code you use the triple backticks to delimit that the next section will be code (formatted differently). 

Is it possible to embed vscode into that spot inside the editor so you can type and get syntax highlighting and code keyboard shortcuts? 

For example, if you created a code block / code block area in the editor, load in vscode in that spot. Then if you tab it creates a tab but doesn't exit the editor. And as you type code autocomplete comes up (for known languages). Does this make sense? 

In other words, in the Github issues editor embed a small vscode instance when the user wants to write code. 

Similarly, if you want to embed the vscode editor in your own webpage you could use an iframe or div and just that area is vscode. Similar to when you would embed YouTube.

**Comment by RedCMD** [comment]
not embedded
but there is vscode web sites
https://vscode.dev/
https://insiders.vscode.dev/
https://github.dev/

you can access by pressing `.` with a code file open in github

**Comment by velara3** [comment]
That's cool!!!
